### ISSUE TITLE
Sheet in Legacy Rules

### ISSUE THREAD

**Issue by Denner09** [foundryvtt/dnd5e#4316]
Switching to legacy rules leaves the sheet like this
![Print1](https://github.com/user-attachments/assets/f5832983-1e36-49fc-a8ed-4c19fd518bf0)
![Print2](https://github.com/user-attachments/assets/a558714a-e346-4886-8a9f-6639ca167a00)

**Comment by aramil88** [comment]
+1

**Comment by JPMeehan** [comment]
Any console errors? Module free replication?

**Comment by thrahistan** [comment]
Same problem here. Created new world for testing if something else made a hickup but it's true for me this way too.
No modules activated.
Console error which generates:
![image](https://github.com/user-attachments/assets/e980dc97-bfac-4fee-9c27-1bb838eb694f)

**Comment by JPMeehan** [comment]
What browsers are you using?

**Comment by Denner09** [comment]
Maybe I've found the problem. The variables are bugging in the legacy mode if the foundry language isn't in english. I redid all the standards steps:
- I created a new world
- I started without any modules
- I changed to Legacy Rules

And the error persisted, but switching back to the default Foundry language made it go back to normal.
![Print3](https://github.com/user-attachments/assets/b295acd1-66d6-4fe3-9f4b-e5861f82ae4b)
